compiler/common-artifacts/src/TestDataGenerator.cpp

```diff
@@ -144,8 +144,8 @@ int entry(int argc, char **argv)
     {
       const auto *input_node = dynamic_cast<const luci::CircleInput *>(node);
       std::string name = input_node->name();
-      if (name.find(":0") == std::string::npos)
-        name += ":0";
+      if (name.find(":") == std::string::npos)
+        name += ":";
```

Comment:
This will make input name "~~:".

Maybe `name += ":" + std::to_string(<index_of_input_node>)` ?